src/pages/Swap/index.tsx

```diff
@@ -140,7 +141,7 @@ export default function Swap({ history }: RouteComponentProps) {
     [independentField, parsedAmount, showWrap, trade]
   )
 
-  const fiatValueInput = useUSDCValue(parsedAmounts[Field.INPUT])
+  const fiatValueInput = useUSDCValue(parsedAmounts[Field.INPUT] ?? pricedAmount)
   const fiatValueOutput = useUSDCValue(parsedAmounts[Field.OUTPUT])
```

Comment:
any reason you aren't applying this treatment to output?

I think if output token is selected, we'd want to show spot price as well

Reply:
ok thanks, that was a question I had.  Can do as well.